algorithms/src/CMakeLists.txt

```diff
@@ -34,5 +34,5 @@ kokkos_lib_include_directories(
 )
 target_link_libraries(kokkosalgorithms PUBLIC kokkoscore)
 
-kokkos_link_tpl(kokkoscontainers PUBLIC ROCTHRUST)
-kokkos_link_tpl(kokkoscore PUBLIC ONEDPL)
+kokkos_link_tpl(kokkosalgorithms PUBLIC ROCTHRUST)
+kokkos_link_tpl(kokkosalgorithms PUBLIC ONEDPL)
```

Comment:
Drive-by. I was initially adding a dependency on `kokkoscontainers` needing me to mess with the dependencies anyway.